resources/views/applicant/static_faq.html.twig

```diff
@@ -15,7 +15,37 @@
 
     {% include "common/goc" %}
 
-    {% include "common/header" with {'header':faq} %}
+    {# {% include "common/header" with {'header':faq} %} #}
+
+    <header data-clone>
+        <div data-c-overlay="black(80)" data-c-padding="tb(triple)" style="background-image: url('/images/sean-pollock.jpg');">
+            <div data-c-container="large">
+                <h1 data-c-color="white" data-c-font-weight="bold" data-c-font-size="h2">Talent Cloud FAQ</h1>
+                <div data-c-alignment="left" data-c-margin="top(double)">
+                    <nav
+                        aria-label="breadcrumb"
+                        data-c-breadcrumbs>
+                        <ol>
+                            <li>
+                                <a
```

Comment:
Was this heading included on purpose?

Reply:
Not sure what you mean. There's a new page header, yes, but it needs to be Reactified, which is why I didn't bother partialing it (it needs to include the Reactified breadcrumbs too).